chrome/fed-modules.json

```diff
@@ -699,5 +699,17 @@
               ]
           }
       ]
+    },
+    "connect": {
```

Comment:
The key should be `statospherePlf`. Again it depends on the appname, but due to some browser env limitations we can't support the dash character `-`. That is why its converted to camelCase